ext/opcache/zend_accelerator_module.c

```diff
@@ -848,7 +848,7 @@ ZEND_FUNCTION(opcache_get_configuration)
 	add_assoc_long(&directives,   "opcache.jit_max_recursive_returns", JIT_G(max_recursive_returns));
 	add_assoc_long(&directives,   "opcache.jit_max_root_traces", JIT_G(max_root_traces));
 	add_assoc_long(&directives,   "opcache.jit_max_side_traces", JIT_G(max_side_traces));
-	add_assoc_long(&directives,   "opcache.jit_prof_threshold", JIT_G(prof_threshold));
+	add_assoc_long(&directives,   "opcache.jit_prof_threshold", (zend_long) JIT_G(prof_threshold));
```

Comment:
Shouldn't this rather be
```suggestion
	add_assoc_double(&directives,   "opcache.jit_prof_threshold", JIT_G(prof_threshold));
```
cc @dstogov

Reply:
This should be a double indeed, not a long

Reply:
Are we fine changing this for PHP-8.3?

Reply:
I am, this is a bug imo

Reply:
After having a closer look, I agree. See #17077.